k8s/namespaces/wa/wa-task-batch-initiation-failure/aat-00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-initiation-failure
+spec:
+  values:
+    job:
+      schedule: "0-59/10 * * * *"
```

Comment:
You probably want a different schedule within each cluster otherwise they'll run at the same time giving you the same result.

Reply:
will change

Reply:
No, it works like following, each cluster works in 10 minutes, one of them starts 00. and other one starts 05. min
aat-00 cluster works 00, 10, 20 ... 50 minutes
aat-01 cluster works 05, 15,25, ... 55  minutes